packages/unit-vitest/src/templates/base/src/components/___examples__/ExampleComponent.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div>
+    <p>{{ title }}</p>
+    <q-list>
+      <q-item v-for="todo in todos" :key="todo.id" @click="increment" clickable>
+        {{ todo.id }} - {{ todo.content }}
+      </q-item>
+    </q-list>
+
+    <p>Count: {{ todoCount }} / {{ meta.totalCount }}</p>
+    <p>Active: {{ active ? 'yes' : 'no' }}</p>
+    <p>Clicks on todos: {{ clickCount }}</p>
+  </div>
+</template>
+
+<script lang="ts" setup>
```

Comment:
I'm not really sold on using setup syntactic sugar, but I guess it's ok to leave these as-is
But we need to do the TS/JS differentiated generation for sure before releasing

Dunno if it's easier by creating different files altogether or by using build-in templating to add/omit TS pieces

Reply:
As far as I know the setup was recommended by Evan to use. I remember that from one of his talks. It might be a matter of preference but I always use setup, as it is less boilerplate code :)

Reply:
I'm not really a fan of scaffolding it by default yet, it feels a bit too magical to me
But as I said, that's fine, I guess we'll eventually all move to use it anyway